src/client/app/redux/actions/map.ts

```diff
@@ -2,24 +2,20 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import {ActionType, Dispatch, GetState, Thunk} from '../../types/redux/actions';
+import { ActionType, Dispatch, GetState, Thunk } from '../../types/redux/actions';
 import * as t from '../../types/redux/map';
```

Comment:
OEd is going away from import *. Having said that, I think a number of the imported items will go away when the new Redux way is used (for example, actions).